pkgs/development/python-modules/biom-format/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cython,
+  click,
+  numpy,
+  scipy,
+  pandas,
+  h5py,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "biom-format";
+  version = "2.1.15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "biocore";
+    repo = "biom-format";
+    rev = "refs/tags/${version}";
+    hash = "sha256-WRBc+C/UWme7wYogy4gH4KTIdIqU3KmBm2jWzGNxGQg=";
+  };
+
+  build-system = [
+    setuptools
+    cython
```

```suggestion
    cython
    numpy
```
numpy is required not only at runtime but also at build time.
https://github.com/biocore/biom-format/blob/2.1.15/pyproject.toml

---

Thanks